content/riak/kv/2.1.4/using/cluster-operations/backing-up.md

```diff
@@ -258,34 +259,20 @@ the following commands on `riak6.example.com`.
     riak-admin cluster commit
     ```
 
-Your [configuration files][config reference] should also be changed to match the new
-name in addition to running the commands (the `-name` setting in
-`vm.args` in the older config system, and the `nodename` setting in
-`riak.conf` in the newer system). If the IP address of any node has
-changed, verify that the changes are reflected in your configuration
-files to ensure that the HTTP and Protocol Buffers interfaces are
-binding to the correct addresses.
-
-A robust DNS configuration can simplify the restore process if the IP
-addresses of the nodes change, but the hostnames are used for the node
-names and the hostnames stay the same. Additionally, if the HTTP and
-Protocol Buffers interface settings are configured to bind to all IP
-interfaces (0.0.0.0), then no changes will need to be made to your
-configuration files.
-
-When performing restore operations involving `riak-admin cluster
-force-replace`, we recommend that you start only one node at a time and
-verify that each node that is started has the correct name for itself
-and for any other nodes whose names have changed.
-
-To do this, first verify that the correct name is present your
-configuration file. Then, once the node is started, run `riak attach` to
-connect to the node. It may be necessary to enter an Erlang atom and
-press enter to obtain a prompt, by typing `x.` and pressing enter. The
-prompt obtained should contain the correct node name. Disconnect from
-the attached session with **Ctrl-G q**. Finally, run `riak-admin
-member_status` to list all of the nodes and verify that all nodes listed
-have the correct names.
+Your [configuration files][config reference] should also be changed to match the new name in addition to running the commands (the `-name` setting in `vm.args` in the older config system, and the `nodename` setting in `riak.conf` in the newer system).
```

Comment:
Cuttlefish allows for vm.args and app.config all the way to current versions.  It's probably worth a discussion about disallowing configuration using the legacy files in the future, but they do currently work.